custom/icds_reports/static/js/directives/download/download.directive.js

```diff
@@ -507,6 +509,10 @@ function DownloadController($rootScope, $location, locationHierarchy, locationsS
         return vm.selectedIndicator === 11;
     };
 
+    vm.isChildGrowthSelected = function () {
```

Comment:
can we call it as growth tracker? So that no one confuses it with growth monitoring list

Reply:
updated!!